### DISCUSSION TITLE
Linux on W100

### DISCUSSION THREAD

**Discussion by gapobroni** [biolds/tibuta-w100#2] [Q&A] [answered]
Hello biolds,
I got few days ago this device and want to use with linux. I tested Lubuntu 22.04.2 but haver serious difficulties to configure it, specialy Wifi. After so many searches I got this document. I would like to ask you some questions.
I am not so deep into linux world but it is the only system I use on my desktop.
Firstly: when you say Debian bookworm are you reffering to the machine it self W100?
Secondly: I do not see the wifi section in your document
Third: I do not understand what to do with the linux-image-6.1.12+_6.1.12+-1_amd64.deb. I am not sure, but I would like to think, I have to install first debian complet system and then run/install this *.deb file.
I would like to confirugre my W100 to be able to use as my second machine. Thank you very much for your kindly help.

**Comment by biolds** [comment, marked as answer]
Hi,

Debian is a Linux based operating system like Ubuntu, Bookworm is the name of the atest reease of Debian.

There is no Wifi section because I could not make it work, I'm using an external Wifi dongle instead.

I have built and tested the kernel ``deb`` file on Debian. Though, you could try to run the commands on Ubuntu, since it is based on Debian there are chances it will work on it.
